src/arch/svm/types.ts

```diff
@@ -36,12 +36,12 @@ export enum SVMEventNames {
 
 export type EventName = keyof typeof SVMEventNames;
 
-export type EventWithData<T extends EventData> = {
+export type EventWithData = {
   confirmationStatus: string | null;
   blockTime: UnixTimestamp | null;
   signature: Signature;
   slot: bigint;
   name: EventName;
-  data: T;
+  data: unknown;
```

Comment:
I don't love getting rid of this type bound, but I don't think we can have this if we want to use the eventsClient to query events for other contracts.